tests/test_pydantic.py

```diff
@@ -13,8 +20,7 @@ class Model(pydantic.BaseModel):
     instance = Model(content=t)
     assert isinstance(instance.content, DataURI)
     assert (
-        instance.json()
-        == '{"content": "data:text/plain;charset=utf-8;base64,VGhlIHF1aWNrIGJyb3duIGZveCBqdW1wZWQgb3ZlciB0'
-        'aGUgbGF6eSBkb2cu"}'
+        instance.__getattr__(func_json)()
+        == '{"content":"data:text/plain;charset=utf-8;base64,VGhlIHF1aWNrIGJyb3duIGZveCBqdW1wZWQgb3ZlciB0aGUgbGF6eSBkb2cu"}'
```

Comment:
could we test both?

Reply:
I don't know how to test for this edge case